src/VideoStreaming/VideoReceiver.cc

```diff
@@ -468,6 +473,65 @@ VideoReceiver::start()
 #endif
 }
 
+#if defined(QGC_GST_STREAMING)
+void
+VideoReceiver::_startAudio()
+{
+    // Run audio
+    GError *error = NULL;
+    uint32_t port = _videoSettings->audioUdpPort()->rawValue().toUInt();
+    QString pipeline = QStringLiteral("udpsrc port=%1 ! application/x-rtp, media=audio, clock-rate=44100, encoding-name=L16, encoding-params=1, channels=1, payload=96 ! rtpL16depay name=rtp0 ! audioconvert ! volume volume=1.0 name=volume0 ! queue ! autoaudiosink sync=false");
```

Comment:
```suggestion
    QString pipeline = QStringLiteral("udpsrc port=%1 ! application/x-rtp, media=audio, clock-rate=44100, encoding-name=L16, encoding-params=1, channels=1, payload=96 ! rtpL16depay name=rtp0 ! audioconvert ! volume volume=1.0 name=volume0 ! queue ! autoaudiosink sync=false").arg(port);
```

Remove arg(port) below